### PR TITLE
Add watchdog mechanism to swss service and generate alert when swss have issue.

### PR DESCRIPTION
**Pull request by liuh-80**
This PR depends on https://github.com/sonic-net/sonic-swss/pull/2737 merge first.

**What I did**
Add orchagent watchdog to monitor and alert orchagent stuck issue.

**Why I did it**
Currently SONiC monit system only monit orchagent process exist or not. If orchagent process stuck and stop processing, current monit can't find and report it.

**How I verified it**
Pass all UT.
Add new UT https://github.com/sonic-net/sonic-mgmt/pull/8306 to check watchdog works correctly.
Manually test, after pause orchagent with 'kill -STOP <pid>', check there are warning message exist in log:

Apr 28 23:36:41.504923 vlab-01 ERR swss#supervisor-proc-watchdog-listener: Process 'orchagent' is stuck in namespace 'host' (1.0 minutes).

**Details if related**
Heartbeat message PR: https://github.com/sonic-net/sonic-swss/pull/2737
UT PR: https://github.com/sonic-net/sonic-mgmt/pull/8306

### CODE REVIEW

File: dockers/docker-orchagent/supervisord.conf.j2 [hunk 1]
```diff
@@ -19,6 +19,13 @@ autostart=true
 autorestart=unexpected
 buffer_size=1024
 
+[eventlistener:supervisor-proc-watchdog-listener]
+command=/usr/bin/supervisor-proc-watchdog-listener --container-name swss
```
Comment (qiluo-msft) [inline dockers/docker-orchagent/supervisord.conf.j2:23]: >listener [](http://example.com/codeflow?start=42&length=8)

Could you explore how much code could be reused if combined this listener with above "supervisor-proc-exit-listener"?

Comment (liuh-80) [reply dockers/docker-orchagent/supervisord.conf.j2]: Fixed, merged code to supervisor-proc-exit-listener

File: dockers/docker-orchagent/supervisord.conf.j2 [hunk 2]
```diff
@@ -75,6 +75,7 @@ command=/usr/bin/orchagent.sh
 priority=4
 autostart=false
 autorestart=false
+stdout_capture_maxbytes=1MB
```
Comment (qiluo-msft) [inline dockers/docker-orchagent/supervisord.conf.j2:78]: >stdout_capture_maxbytes [](http://example.com/codeflow?start=0&length=23)

What is the reason of this change?

**Review by qiluo-msft — APPROVED** [review]
Approved, with an open question in the comment.

**Review by qiluo-msft — APPROVED** [review]
Approved, with an open question in the comment.

File: dockers/docker-orchagent/supervisord.conf.j2 (hunk 2 above)
Comment (liuh-80) [reply dockers/docker-orchagent/supervisord.conf.j2:78]: This config will enable stdout capture on orchagent, then systemd will convert orchagent heartbeat message to systemd PROCESS_COMMUNICATION_STDOUT event.
